.github/workflows/test.yml

```
@@ -0,0 +1,32 @@
+name: Test
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)